### PR TITLE
Improve quickstart dependencies install script

### PR DESCRIPTION
**Pull request by grzesiek**
### Description

This PR adds a few minor changes to quick start dependencies installation script:

1. It adds convenience commands to move `golang-migrate` and `comby` into `/usr/local/bin`
2. It fixes `cp` commands used to move `watchman` binary and libraries into `/usr/local/{bin, lib}` that were broken.
3. It removes the command that creates `/usr/local/{bin,lib}` as these directories should be always available on a fresh Ubuntu installation (including `ubuntu:latest` Docker container).

### CODE REVIEW

File: doc/dev/getting-started/quickstart_1_install_dependencies.md [hunk 1]
```diff
@@ -165,16 +165,20 @@ The following are two recommendations for installing these dependencies:
 
     # install golang-migrate (you must rename the extracted binary to `golang-migrate` and move the binary into your $PATH)
```
Comment (mrnugget) [inline doc/dev/getting-started/quickstart_1_install_dependencies.md:166]: This comment is now outdated. I suggest moving what's in the parentheses down above the `mv` command you added and changing it do something like this:

```
# The extracted binary must be in your $PATH. Here's how you'd move it to `/usr/local/bin` (which is most likely in your `$PATH`):
```

**Review by mrnugget — CHANGES_REQUESTED** [review]
Thanks! Left a comment because I think the comments in two places are now outdated.

**Review by mrnugget — APPROVED** [review]
Fantastic! Thanks

**Comment by grzesiek** [comment]
> Fantastic! Thanks

@mrnugget Is there anything I need to do to get this merged?

**Comment by mrnugget** [comment]
> @mrnugget Is there anything I need to do to get this merged?

I'm investigating why CI doesn't run. Other than that: nothing to do from your side.

**Comment by mrnugget** [comment]
This page in the handbook describes it: https://about.sourcegraph.com/handbook/engineering/external_contributions

Kicked off the build. When that's green, I can merge it.

**Pull request merged**